templates/form/input--search.html.twig

```diff
@@ -13,22 +13,25 @@
  * @see ./core/themes/stable/templates/form/input.html.twig
  */
 #}
-
-{% include '@ecl-twig/form-group' with {
-  input: {
-    input_type: 'search',
-    icon_path: ecl_icon_path,
-    id: input_array.id,
-    name: input_array.name,
+{% if element['#id'] != 'admin-toolbar-search-input' %}
```

Comment:
As I understand solution already suggested and it works: https://github.com/openeuropa/oe_theme/issues/1176#issuecomment-1268392745